src/test/java/posters/pageobjects/pages/checkout/NewShippingAddressPage.java

```diff
@@ -41,9 +41,10 @@
 
     @Override
     @Step("ensure this is a new shipping address page")
-    public void isExpectedPage()
+    public NewShippingAddressPage isExpectedPage()
     {
         headline.should(exist);
```

Comment:
Please call `super.isExpectedPage();` at first in case we would add some default functionality.

Reply:
@ufjena this hasn't been covered yet.